dev/vm_printers.ml

```diff
@@ -78,7 +77,7 @@ and ppwhd whd =
   | Vatom_stk(a,s) ->
       open_hbox();ppatom a;close_box();
       print_string"@";ppstack s
-  | Vuniv_level lvl -> Feedback.msg_notice (Univ.Level.pr lvl)
+  | Vuniv_level lvl -> Feedback.msg_notice (Univ.Level.pr lvl) (* FIXME *)
```

Comment:
Could you make it clear what the FIXME is about in the comment? I guess this one is about the msg_notice usage.

Reply:
Yeah, I found it strange while reading the code, but it is orthogonal to this PR. Will improve the comment.